themes/gauge_theme/layout.html

```diff
@@ -87,40 +87,22 @@
 
 {%- block content %}
 <header class="top">
-  <div class="container">
-    <a class="logo" href="//gauge.org">
-      <img alt="Gauge logo" id="thumb" src="{{ pathto('_static/img/Gauge-Logo-White.svg', 1) }}" width="150" height="38">
-    </a>
-    <span class="beta">Preview</span>
-    <nav class="mainnav">
-      <ul class="level0">
-        <li class="dropdown">
-        <a href="//gauge.org/get-started.html" class="link-get-started">Get Started</a>
-        <ul class="level1">
-        <li>
-        <a href="//gauge.org/get-started.html" class="link-download">Download</a>
-        </li>
-        <li>
-        <a href="//gauge.org/plugins.html" class="link-plugins">Plugins</a>
-        </li>
-        </ul>
-        </li>
-        <li class="dropdown">
-        <a href="/" class="link-documentation">Documentation</a>
-        <ul class="level1">
-            <li> <a href="{{ pathto('examples') }}">Examples</a> </li>
-            <li> <a href="{{ pathto('writing-specifications') }}">Write Specifications</a> </li>
-            <li> <a href="{{ pathto('execution') }}">Run Specifications</a> </li>
-            <li> <a href="{{ pathto('configuration') }}">Configuration</a></li>
-            <li> <a href="{{ pathto('faqs') }}">FAQs</a></li>
-            <li> <a href="//manpage.gauge.org">Manual</a></li>
-          </ul>
-        </li>
-        <li>
-        <a href="//blog.getgauge.io" class="link-blog">Blog</a>
+  <div class="header-container">
+    <a href="//gauge.org" class="logo">Gauge</a>
+    <nav class="nav">
+      <ul class="main-nav">
+        <li class="githubstar-box">
+          <a href="https://github.com/getgauge/gauge" target="_blank"><span  class="github-icon"></span><span class="github_star"></span></a>
         </li>
+        <li class="link_home"><a href="//gauge.org">Home</a></li>
+        <li class="link_plugins"><a href="//gauge.org/plugins.html">Plugins</a></li>
+        <li><a href="https://docs.gauge.org/latest/index.html">Documentation</a></li>
```

Comment:
can we make the url's consistent?
Please use the form `href="//gauge.org/get-started.html`